quantum/rgb_matrix_animations/solid_reactive_simple_anim.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+#ifdef RGB_MATRIX_KEYPRESSES
+#ifndef DISABLE_RGB_MATRIX_SOLID_REACTIVE_SIMPLE
+
+extern rgb_config_t rgb_matrix_config;
+extern last_hit_t g_last_hit_tracker;
+
+bool rgb_matrix_solid_reactive(bool init, uint8_t iter) {
```

Comment:
```suggestion
bool rgb_matrix_solid_reactive_simple(bool init, uint8_t iter) {
```

Reply:
hmm, not sure how I screwed that up, but thanks for catching it.